apps/src/gamelab/AnimationPicker/actions.js

```diff
@@ -12,9 +12,11 @@ var AnimationPickerAction = module.exports.AnimationPickerAction = {
  * Show the animation picker
  * @returns {{type: AnimationPickerAction}}
  */
-module.exports.showAnimationPicker = function () {
+module.exports.showAnimationPicker = function (onComplete, onCancel) {
   return {
-    type: AnimationPickerAction.SHOW_ANIMATION_PICKER
+    type: AnimationPickerAction.SHOW_ANIMATION_PICKER,
+    onComplete: onComplete,
+    onCancel: onCancel
```

Comment:
I think the main problem with storing the callback in your store is you've lost serializability. Writing "universal" apps, where you can have a reducer run on the client and server, and the server can easily hydrate the client is an important use case for redux in general that you lose out on if you can serialize.

It also makes some debugging operations more difficult. For example, an approach that is often used is to regularly save the state to localStorage, such that when I reload the page, I can reload that state and be where I was before reloading.

I'm not sure either of these cases are particuarly important to our own needs.